nss-day-cohort-48/debugging-assessment-thejmdw
# kids names rendering

To test:

1. Pull code
2. git checkout thejmdw-second-commit
3. open in VScode
4. serve and open in Chrome
5. check to see that kids names are being rendered to the DOM
